.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
```

Comment:
People who wanted a quick reply could already have used Gitter, as it's listed on the Julia webpage. This template is intended at people who would have filed an issue on GitHub. For this use case, creating a new post makes more sense than going to Gitter.

We could point people to the "Community" page to let them choose the best channel. For example, they could choose a non-English forum.